vendor/bat-native-ads/src/bat/ads/internal/static_values.h

```diff
@@ -31,6 +31,28 @@ static const uint64_t kNextEasterEggStartsInSeconds = 30;
 
 static char kShoppingStateUrl[] = "https://amazon.com";
 
+static std::map<std::string, std::vector<std::string>> kShoppingSiteCategories = {
+  {"cars", {
+    "https://www.carmax.com", "https://www.cargurus.com",
```

Comment:
Should we not consider adding this information to the catalog we download?

Reply:
kCampaignCheckoutUrls and kSegmentKeywords should ultimately be defined by marketers so probably better be served from the catalog. The kShoppingSiteCategories data structure could evolve into something that holds more sites (100s/1000s) with maybe meta data about those sites and structure to infer shopping state. Maybe the catalogue also makes sense for this.